magento/magento2
# #26850 Adjusted configurable product price indexing to save all values based on available simple options

### Description (*)

This is not final solution to the linked ticket - this PR aims to provide correct data in `catalog_product_index_price` for configurable products, so that data might be used later i.e. on products listing.

### Related Pull Requests


### Fixed Issues (if relevant)

1. Fixes (partially) magento/magento2#26850

### Manual testing scenarios (*)

1. Create configurable product with simple child products
2. Reindex `catalog_product_price` indexer
3. Verify configurable product prices in `catalog_product_index_price` table - all values should be correctly calculated from child products data

### Questions or comments


### Contribution checklist (*)
